server/routes/about.js

```diff
@@ -59,4 +59,17 @@ router.post('/email', async (req, res) => {
   }
 });
 
+router.post('/customer-service', async (req, res) => {
+  const { name, email, subject, message } = req.body;
+  const done = await sendSlackFeedbackMessage(name, email, subject, message);
+
+  if (done) {
+    return res.status(200).send({
```

Comment:
we should do toasts for all success msgs in the future? @gnorbsl @Enubia @Tokikko - did someone already work on this?

Reply:
We could,maybe just make the toast thing configurable so that I can show different colors based on what you want to show